qos-service-impl/src/main/java/com/palantir/atlasdb/qos/metrics/QosMetrics.java

```diff
@@ -61,4 +73,17 @@ public void recordWrite(QueryWeight weight) {
         rowsWritten.mark(weight.numDistinctRows());
     }
 
+    public void recordBackoffMicros(long backoffTimeMicros) {
+        if (backoffTimeMicros > 0) {
+            log.info("Backing off for {} micros", SafeArg.of("backoffTimeMicros", backoffTimeMicros));
+            backoffTime.mark(backoffTimeMicros);
```

Comment:
This could make strange graphs if we only mark when backing off.

Reply:
I think we'd look at the derivative or m1/m5 rates for this graph (it's basically a indicator of whether or not you're currently getting rate limited, and by how much)